cmd/deploy/remote.go

```diff
@@ -199,7 +199,6 @@ func (rd *remoteDeployCommand) deploy(ctx context.Context, deployOptions *Option
 	if sshSock != "" {
 		if _, err := os.Stat(sshSock); err != nil {
 			oktetoLog.Debugf("Not mounting ssh agent. Error reading socket: %s", err.Error())
-			sshSock = ""
```

Comment:
Why is this a ineffassign? `If sshSock != ""` then we should set it as ="". I can't figure out why is wrong

Reply:
sshShock is only used at the else clause (line 204) - this assignement of new value is never read

Reply:
Thanks for explaining it. I wasn't able to get it from the GH interface. 😄